include/wasm/chaining.h

```diff
@@ -8,12 +8,24 @@ namespace wasm {
 
 int awaitChainedCall(unsigned int messageId);
 
+int awaitChainedCall2(unsigned int messageId);
```

Comment:
These functions really need some explanation, what's the use for `awaitChainedCall2`? How is it different to `awaitChainedCall`?